New version: FastCopy ver 4.0.1

matcha-shake opened the pull request:
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----


###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/41853)

matcha-shake committed:
Add new ver: 4.0.1's manifest file

FastCopy ver: 4.0.1 was released at 2022-01-24 (JST).
I created a manifest file to support this update.

matcha-shake committed:
Fix: PackageVersion

matcha-shake committed:
Update and rename FastCopy.FastCopy.installer.yaml to FastCopy.FastCopy.yaml

matcha-shake committed:
Update FastCopy.FastCopy.yaml

Trenly commented on manifests/f/FastCopy/FastCopy/4.0.1/FastCopy.FastCopy.yaml:
```diff
@@ -0,0 +1,24 @@
+PackageIdentifier: FastCopy.FastCopy
+PackageVersion: 4.0.1
+PackageName: FastCopy
+Publisher: H.Shirouzu
```
The Publisher needs to match what is in control panel
```suggestion
Publisher: H.Shirouzu & FastCopy Lab, LLC.
```

matcha-shake replied:
Thank you for the suggestion.
I update the file based on the software license page.
https://fastcopy.jp/help/fastcopy_eng.htm#license

Trenly commented on manifests/f/FastCopy/FastCopy/4.0.1/FastCopy.FastCopy.yaml:
```diff
@@ -0,0 +1,24 @@
+PackageVersion: 4.0.1
+PackageName: FastCopy
+Publisher: H.Shirouzu
+License: GPL 3.0
```
```suggestion
License: GPL-3.0
```

matcha-shake replied:
Thank you for the suggestion.
I update the file based on the software license page.
https://fastcopy.jp/help/fastcopy_eng.htm#license

matcha-shake committed:
Update FastCopy.FastCopy.yaml

Update "Publisher"
Update "License" information

matcha-shake committed:
Fix FastCopy.FastCopy.yaml

Fixed for the following error:
Manifest Error: All field names should be PascalCased. Field: LicenseURL Line: 9, Column: 1

ItzLevvie reviewed (changes requested):
Package was merged: https://github.com/microsoft/winget-pkgs/blob/master/manifests/f/FastCopy/FastCopy/4.0.1/FastCopy.FastCopy.yaml

matcha-shake commented:
Closed. Because the commit created by the FastCopy's auther has been merged.
